src/test/java/kuit/subway/utils/DatabaseCleanup.java

```diff
@@ -34,10 +34,25 @@ public void execute() {
         entityManager.createNativeQuery("SET REFERENTIAL_INTEGRITY FALSE").executeUpdate();
 
         for (String tableName : tableNames) {
-            entityManager.createNativeQuery("TRUNCATE TABLE " + tableName).executeUpdate();
-            entityManager.createNativeQuery("ALTER TABLE " + tableName + " ALTER COLUMN ID RESTART WITH 1").executeUpdate();
+            String primaryKeyColumnName = determinePrimaryKeyColumnName(tableName);
+
+            if (primaryKeyColumnName != null) {
+                entityManager.createNativeQuery("TRUNCATE TABLE " + tableName).executeUpdate();
+                entityManager.createNativeQuery("ALTER TABLE " + tableName + " ALTER COLUMN " + primaryKeyColumnName + " RESTART WITH 1").executeUpdate();
```

Comment:
질문에 대한 답으로 저라면 이렇게 할 것 같아요!
```suggestion
            entityManager.createNativeQuery("TRUNCATE TABLE " + tableName).executeUpdate();
            entityManager.createNativeQuery("ALTER TABLE " + tableName + " ALTER COLUMN" + tableName + "_id" + " RESTART WITH 1").executeUpdate();
```

Reply:
배워갑니다-!